CHANGELOG

```diff
@@ -1,10 +1,41 @@
 v3.9.0 (XXXX-XX-XX)
 -------------------
 
+* Changed various default values for RocksDB to tune operations for different
+  typical scenarios like gp2 type volumes and gp3 type volumes and locally
+  attached SSDs with RAID0:
+  - `--rocksdb.level0-slowdown-trigger` has been decreased from 20 to 16
```

Comment:
we should mention the change for the hardware acceleration here too!

Reply:
Indeed, will do.

Reply:
Done. I will not restart Jenkins for this.,